crates/uv/tests/pip_list.rs

```diff
@@ -460,3 +460,298 @@ fn exclude() -> Result<()> {
+        .into_iter()
+        .chain(INSTA_FILTERS.to_vec())
+        .collect::<Vec<_>>();
+
+    // Install the editable package.
+    uv_snapshot!(filters, Command::new(get_bin())
+        .arg("pip")
+        .arg("install")
+        .arg("-e")
+        .arg("../../scripts/editable-installs/poetry_editable")
+        .arg("--strict")
+        .arg("--cache-dir")
+        .arg(context.cache_dir.path())
+        .arg("--exclude-newer")
+        .arg(EXCLUDE_NEWER)
+        .env("VIRTUAL_ENV", context.venv.as_os_str())
+        .env("CARGO_TARGET_DIR", "../../../target/target_install_editable"), @r###"
+    success: true
+    exit_code: 0
+    ----- stdout -----
+
+    ----- stderr -----
+    Built 1 editable in [TIME]
+    Resolved 2 packages in [TIME]
+    Downloaded 1 package in [TIME]
+    Installed 2 packages in [TIME]
+     + numpy==1.26.2
+     + poetry-editable==0.1.0 (from file://[WORKSPACE_DIR]/scripts/editable-installs/poetry_editable)
+    "###
+    );
+
+    // Account for difference length workspace dir
+    let prefix = if cfg!(windows) { "file:///" } else { "file://" };
+
+    let workspace_len_difference = workspace_dir.as_str().len() + 32 - 16 - prefix.len();
+    let find_divider = "-".repeat(25 + workspace_len_difference);
+    let replace_divider = "-".repeat(57);
+
+    let find_header = format!(
+        "Editable project location{0}",
+        " ".repeat(workspace_len_difference)
+    );
+    let replace_header = format!("Editable project location{0}", " ".repeat(32));
+
+    let find_whitespace = " ".repeat(25 + workspace_len_difference);
+    let replace_whitespace = " ".repeat(57);
+
+    let search_workspace = workspace_dir.as_str().strip_prefix(prefix).unwrap();
+    let replace_workspace = "[WORKSPACE_DIR]/";
+
+    let filters = INSTA_FILTERS
+        .iter()
+        .copied()
+        .chain(vec![
+            (search_workspace, replace_workspace),
+            (find_divider.as_str(), replace_divider.as_str()),
+            (find_header.as_str(), replace_header.as_str()),
+            (find_whitespace.as_str(), replace_whitespace.as_str()),
+        ])
+        .collect::<Vec<_>>();
+
+    uv_snapshot!(filters, Command::new(get_bin())
+    .arg("pip")
+    .arg("list")
+    .arg("--format=freeze")
+    .arg("--cache-dir")
+    .arg(context.cache_dir.path())
+    .env("VIRTUAL_ENV", context.venv.as_os_str())
+    .current_dir(&context.temp_dir), @r###"
+    success: true
+    exit_code: 0
+    ----- stdout -----
+    numpy==1.26.2
+    poetry-editable==0.1.0
+
+    ----- stderr -----
+    "###
+    );
+
+    uv_snapshot!(filters, Command::new(get_bin())
+    .arg("pip")
+    .arg("list")
+    .arg("--format=freeze")
+    .arg("--editable")
+    .arg("--cache-dir")
+    .arg(context.cache_dir.path())
+    .env("VIRTUAL_ENV", context.venv.as_os_str())
+    .current_dir(&context.temp_dir), @r###"
+    success: true
+    exit_code: 0
+    ----- stdout -----
+    poetry-editable==0.1.0
+
```

Comment:
I think most of _freeze_ output format's usecase is to export and re-install current packages in new environment (!?)

Do you know what's usecase of showing version of editable instead of url?

Could we have `pip freeze` behavior on here and show url? Like:
`-e "../../scripts/editable-installs/poetry_editable"`

Reply:
Users will expect uv pip list to behave the same as pip list, so unless there is good reason I would not deviate.

Perhaps there are relevant discussions on the pip repo?

Reply:
> Users will expect uv pip list to behave the same as pip list, so unless there is good reason I would not deviate.

Actually, I don't see reasons and usacases of current `--format freeze` doesn't show urls.
Instead of making overhead, I prefer `list --format freeze` take precedence of `freeze` command over time.


> Perhaps there are relevant discussions on the pip repo?

Yes, there should. needs investigation.

Also, I think it's good to considering this as part of #2023

Reply:
One resource to check if users rely on the current behaviour could be:
https://grep.app/search?q=pip%20freeze